sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/CryptographyClientBuilder.java

```diff
@@ -73,6 +73,7 @@
     HttpLogOptions httpLogOptions;
     final RetryPolicy retryPolicy;
     Configuration configuration;
+    ServiceVersion version;
```

Comment:
All these fields have to be `private`.

Reply:
These same fields are accessed [here](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/KeyEncryptionKeyClientBuilder.java#L71) in the `KeyEncryptionKeyClientBuilder` and so cannot be made private.

Reply:
It is preferable to have package-private getters instead of making the fields package-private.